crates/runtime/src/lib.rs

```diff
@@ -17,12 +17,7 @@ pub mod runtime;
 /// wasmCloud I/O functionality
 pub mod io;
 
-#[deprecated(
-    since = "0.3.1",
-    note = "ActorConfig has been renamed to ComponentConfig"
-)]
-pub use actor::Config as ActorConfig;
-pub use actor::{Component, ComponentInstance, Config as ComponentConfig};
+pub use component::{Component, ComponentInstance, Config as ComponentConfig};
```

Comment:
Here I removed the import of Config as ActorConfig because it was marked deprecated and I didn't find a reference to ActorConfig. Does it seem legit to you?

Reply:
Yeah I think you're good here. The runtime crate hasn't actually been released, so if it isn't anywhere in the repo, we should be good